styles.css

```diff
@@ -825,23 +824,35 @@ form {
     border-color: #01f;
   }
 
+  button {
```

Comment:
[OPTIONAL]Kindly remove the unused code from your work so as to make you code base neat and clean. 👍 😊